src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -100,10 +100,23 @@ class AssimuloFMIAlgOptions(OptionBase):
             solver.
             Default: "Default"
 
+        dynamic_diagnostics --
+            If True, will enable logging of diagnostics data and requires that
+            the option 'result_handling' is set to 'binary', otherwise an exception is raised.
+            If True, the debug file will not be generated.
+            Before simulation start, a check for name clashes between
+            the names of diagnostic variables and model variables is also performed,
+            and an exception is raised if name clashes exist.
+            The diagnostics data will be available via the simulation resuls and/or the
+            binary result file generated during simulation.
+
+            Default: False
+
         logging --
             If True, creates a logfile from the solver in the current
             directory and enables logging of diagnostics data to logfile or resultfile,
             based on simulation option 'result_handling'.
+            If True and 'result_handling' is set to 'binary', a debug file is not generated.

```

Comment:
Do not use the name debug file, the log file from the FMU is still generated and could be mixed up with this one.